src/hardware/mixer.cpp

```diff
@@ -2208,7 +2222,7 @@ class MIXER final : public Program {
 		                           const std::string &xfeed,
 		                           const std::string &reverb,
 		                           const std::string &chorus) {
-			WriteOut("%-22s %4.0f:%-4.0f %+6.2f:%-+6.2f  %-8s %5s %7s %7s\n",
+			WriteOut(MSG_Get("SHELL_CMD_MIXER_HEADER_DELINEATION"),
```

Comment:
Yeah, so I'd append the newline here at the end of the message, or better yet, introduce a `WriteOut` variant that always emits a newline as well at the end.

Reply:
> `WriteOut` variant that always emits a newline as well at the end.

Will do!

Reply:
Turns our WriteOut with various new-line(s) are pretty common:

![2022-09-07_16-57](https://user-images.githubusercontent.com/1557255/189006002-a9861cb8-7839-4737-b912-de8fe7c69b88.png)

So I switched that that approach: `WriteOut("%s\n", MSG_Get(...))`.

Reply:
I know it's not relevant on this, but it seems to me that those IPXNET-related messages are not present in the en.txt file and therefore they are not translatable... Or are they console messages?